contracts/mocks/MockSablierV2LockupLinear.sol

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.19;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../interfaces/sablier/ISablierV2LockupLinear.sol";
+import {LockupLinear} from "../interfaces/sablier/LockupLinear.sol";
+
+contract MockSablierV2LockupLinear is ISablierV2LockupLinear {
```

Comment:
Not quite `Mock` lol - this is the whole implementation 🤣

Reply:
the AI went a little overboard when I asked it to create a mock contract lmao